lib/parsers/package-lock-parser.ts

```diff
@@ -2,7 +2,7 @@ import * as _cloneDeep from 'lodash.clonedeep';
 import * as _isEmpty from 'lodash.isempty';
 import * as _set from 'lodash.set';
 import * as _toPairs from 'lodash.topairs';
-import * as graphlib from '@snyk/graphlib';
+import * as graphlib from 'graphlib';
```

Comment:
did we omit the snyk prefix from the graphlib?

Reply:
https://github.com/snyk/graphlib/blob/master/package.json#L2 :thinking:

Reply:
We don't want to use our fork anymore - it was only swapping a dependency for our lodash fork https://github.com/dagrejs/graphlib/compare/master...snyk:master